docker/gcp_pudl_etl.sh

```diff
@@ -124,7 +124,7 @@ function zenodo_data_release() {
 function notify_slack() {
     # Notify pudl-deployment slack channel of deployment status
     echo "Notifying Slack about deployment status"
-    message="# `${BUILD_ID}` status\n\n"
```

Comment:
markdown wasn't working, and backtics mean something in the shell, so I removed.

Reply:
Whee right this is why I used single quotes in the first place. RIP